tests/commands/test_plugins.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from click.testing import CliRunner
+from tutor.commands.plugins import *
+from tests.test import CONTEXT
+
+
+class PluginsTests(unittest.TestCase):
+    def test_plugins_help(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(plugins_command, ["--help"])
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
+
+    def test_plugins_printroot(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(plugins_command, ["printroot"], obj=CONTEXT)
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
+        self.assertTrue(result.output)
+
+    def test_plugins_list(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(plugins_command, ["list"], obj=CONTEXT)
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
+        self.assertFalse(result.output)
```

Comment:
This statement is failing on my machine because I have many plugins:

```
======================================================================
FAIL: test_plugins_list (commands.test_plugins.PluginsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/regis/projets/overhang/repos/overhang/tutor/tests/commands/test_plugins.py", line 27, in test_plugins_list
    self.assertFalse(result.output)
AssertionError: 'cairn==12.0.11 (disabled)\ndiscovery==12.0.4 (disabled)\necommerce==12.0.2 (disabled)\nforum==12.0.0 (disabled)\nlicense==12.0.1 (disabled)\nmachine==12.0.1 (disabled)\nmfe==12.1.0 (disabled)\nminio==12.0.1 (disabled)\nmonitor==12.0.2 (disabled)\nnotes==12.0.0 (disabled)\nrichie==12.0.0 (disabled)\nstacktracer==0.1.0 (disabled)\nwebui==12.0.1 (disabled)\nxqueue==12.1.0 (disabled)\n' is not false

----------------------------------------------------------------------
```

So in order for this test to pass we should mock the `plugins.iter_installed` function such that it returns an empty list.